wrappers/javascript/package.json

```diff
@@ -46,6 +46,6 @@
     "typescript": "~4.9.4"
   },
   "engines": {
-    "node": ">= 12"
+    "node": ">= 18"
```

Comment:
I don't think that the root workspace package.json engine influences anything when a user installs `@hyperledger/indy-vdr-nodejs`. I think it should be done in the `indy-vdr-nodejs` package only as well. The shared and react native packages can be used with lower node versions.

Reply:
Right, actually the restriction will come from the dependencies (patched `ref-napi` and `ffi-napi`) when attempting to do `yarn install`. Anyway, I'll move it to `indy-vdr-nodejs` to make it clear from there.